openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -26,6 +26,12 @@ <h1>$_('Community Edit Requests')</h1>
     $desc <a href="$href">$link_text</a>
   </div>
 
+</div>
```

Comment:
```suggestion
```
This will close the entire view's `div`, which isn't what we want here.

Reply:
Ah I am so sorry!